src/localstorage.ts

```diff
@@ -1,7 +1,8 @@
+import {eventHandlingMixedIn} from './eventhandling-new';
+import {localStorageAvailable} from './util';
+import {CachingLayer} from './cachinglayer-new';
```

Comment:
Just a formatting thing (which isn't linted at the moment): let's use spaces between curly braces and their content, same as before.